scripts/commit_classifier.py

```diff
@@ -29,10 +29,7 @@ def __init__(self, cache_root):
 
         if not os.path.exists("regressormodel"):
             download_check_etag(URL, "regressormodel.zst")
-            dctx = zstandard.ZstdDecompressor()
-            with open("regressormodel.zst", "rb") as input_f:
-                with open("regressormodel", "wb") as output_f:
-                    dctx.copy_stream(input_f, output_f)
+            decompress("regressormodel.zst")
```

Comment:
In other places you are passing the path without `zst`, here you are passing the path with `zst`. One of the two is wrong